### ISSUE TITLE
19. add roles to background images

### ISSUE THREAD

**Issue by neener** [Simone-Subal-Gallery/simone-subal#21]
Background images that are not just for decoration purposes and should have the same treatment as standard images and include a "role" attribute that equals to "img" and an alternative text description in an "aria-label" attribute.

3-4 hrs
